src/main/java/tutorlink/grade/Grade.java

```diff
@@ -0,0 +1,26 @@
+package tutorlink.grade;
+
+import tutorlink.component.Component;
+import tutorlink.student.Student;
+
+public class Grade {
+    private double score;
+    private Component component;
+    private Student student;
+
+    public Grade(Component component, Student student, double score) {
+        this.component = component;
+        this.student = student;
+        this.score = score;
+    }
+
```

Comment:
Additionally, could you implement getGradeMatricNumber(): String and getGradeComponentDescription(): String as I am unable to use the .equals() override in Grade as I cannot instantiate an instance of the Component abstract class. Thanks!